iotedgedev/cli.py

```diff
@@ -279,11 +279,37 @@ def deploy(manifest_file):
                     "The condition is based on device twin tags or device twin reported properties and should match the expression format. "
                     "For example, tags.environment='test' and properties.reported.devicemodel='4000x'. "
                     "This property can be set via this parameter or in the .env under 'IOTHUB_DEPLOYMENT_TARGET_CONDITION'"))
+@click.option("--device-tag",
+              "-dt",
+              "device_tag",
```

Comment:
Is this line redundant?

Reply:
Your right, but so is it a few lines up for the `target_condition`. Will remove that one as well